test/common.test.ts

```diff
@@ -0,0 +1,52 @@
+import { matchesBlackList } from '../src/common';
+
+export interface TestSetup {
+  givenData: string;
+  shouldMatch: boolean;
+}
+
+describe('alarmName pattern regex match', () => {
+  process.env.IGNORED_ALARM_NAME_PATTERN =
+    '(-AlarmHigh|-AlarmLow|-ProvisionedCapacityHigh|-ProvisionedCapacityLow)-(\\{){0,1}[0-9a-fA-F]{8}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{12}(\\}){0,1}';
+
+  const examples: TestSetup[] = [
+    {
+      givenData:
+        'TargetTracking-table/ci-crawl-status-check-AlarmHigh-03b775c2-ea43-4b01-abe4-2bb1f7c71b30',
+      shouldMatch: true,
+    },
+    {
+      givenData:
+        'TargetTracking-table/ci-crawl-status-check-AlarmLow-0371c1e1-1ea2-4d87-92d1-fdeaeab799a9',
+      shouldMatch: true,
+    },
+    {
+      givenData:
+        'TargetTracking-table/ci-crawl-status-check-ProvisionedCapacityLow-5c3e4c6d-04db-4432-b8bb-c89d3acc8f30',
+      shouldMatch: true,
+    },
+    {
+      givenData:
+        'TargetTracking-table/ci-crawl-status-check-ProvisionedCapacityHigh-36d74b4f-f037-40e0-9207-27a9e986ddc6',
+      shouldMatch: true,
+    },
+    {
+      givenData: 'TargetTracking-table/ci-crawl-status-check-36d74b4f-f037-40e0-9207-27a9e986ddc6',
+      shouldMatch: false,
+    },
+    {
+      givenData:
+        'TargetTracking-table/ci-crawl-status-check-ProvisionedCapacityHigh-36d74b4f-f037-40e0-9207-27a9e86ddc6',
+      shouldMatch: false,
+    },
+    {
+      givenData:
+        'TargetTracking-table/ci-crawl-status-check-ProvisionedCapacityHigh-36d74b4f-f037-40e0-gggg-9207-27a9e86ddc6',
```

Comment:
why doesn't this one match?

Might be useful to add a `description` to each example - even if it's not used (or add it to the test name)

Reply:
Because it's got gggg in the guid. guid pattern are hex characters

Reply:
Done